lib/Dialect/FIRRTL/Transforms/LowerLayers.cpp

```diff
@@ -727,10 +719,12 @@ void LowerLayersPass::runOnOperation() {
   OpBuilder builder(circuitOp);
   SmallVector<std::pair<LayerOp, StringAttr>> layers;
   StringRef circuitName = circuitOp.getName();
+
   circuitOp.walk<mlir::WalkOrder::PreOrder>([&](LayerOp layerOp) {
     auto parentOp = layerOp->getParentOfType<LayerOp>();
     while (parentOp && parentOp != layers.back().first)
       layers.pop_back();
+
```

Comment:
Mega-nits: unrelated whitespace changes. No problem landing anything like this separately on `main`, though!